### PR TITLE
Kk aws fixes dynamo

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md [hunk 1]
```diff
@@ -34,21 +34,101 @@ After you install the connector, you can find it in the **App Explorer**, in the
...
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/addloop.png" alt="The microflow with the loop added">}}
+
+29. Double-click the **Loop** activity.
+30. In the **Iterate over** list, select **ListTableList**.
+
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/editloop.png" alt="Selecting the entity to iterate over">}}
+
+31. In the **Toolbox** pane, search for the **Create object** activity and drag it inside the loop area.
+
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/createobject.png" alt="The microflow with the Create activity added">}}
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws/aws-dynamodb.md:143]: Could the line on the canvas be aligned so there is no bend in it?

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md [hunk 2]
```diff
@@ -34,21 +34,101 @@ After you install the connector, you can find it in the **App Explorer**, in the
...
+24. Double-click the **Create list** activity.
+25. In the **Entity** section, click **Select**.
+26. In the **Select Entity** dialog box, select the entity that you previously added to your domain model, for example, `DBTable`.
+
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/selectentity.png" alt="Selecting the entity">}}
+
+27. In the **Toolbox** pane, search for the **Loop** activity and drag it onto the microflow area.
+28. Position the **Loop** activity before the microflow end event.
+
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/addloop.png" alt="The microflow with the loop added">}}
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws/aws-dynamodb.md:134]: Could the line on the canvas be aligned so there is no bend in it?

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md [hunk 3]
```diff
@@ -34,21 +34,101 @@ After you install the connector, you can find it in the **App Explorer**, in the
...
+    The parameters that you need to configure depend on the contents of the response that an activity expects. The **ListTables** activity used in this example only expects the database table name as a response. Other activities require different parameters. For more information, see [Activities](#activities).
+7. In the **App Explorer**, right-click on the name of your module, and then click **Add microflow**.
+    
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/addmicroflow.png" alt="Adding a microflow">}}
+
+8. Enter a name for your microflow, for example, *ACT_ListTables*, and then click **OK**.
+9. In the **App Explorer**, in the **Amazon DynamoDBConnector** > **Operations** section, find the **ListTables** activity.
+10. Drag the **ListTables** activity onto the work area of your microflow.
+  
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/listtables.png" alt="The ACT_ListTables microflow with the ListTables activity">}}
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws/aws-dynamodb.md]: Could the line on the canvas be aligned so there is no bend in it?

**Review by brfreekmx — CHANGES_REQUESTED** [review]

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-dynamodb.md:134]: Done.

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md (hunk 3 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-dynamodb.md]: Done.

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-dynamodb.md:143]: Done.

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md [hunk 4]
```diff
@@ -34,21 +34,101 @@ After you install the connector, you can find it in the **App Explorer**, in the
...
+    2. Decide whether you want to use session or static credentials to authenticate.
+        The DynamoDB connector supports both session and static credentials. By default, the connector is pre-configured to use static credentials, but you may want to switch to session credentials, for example, to increase the security of your app. For an overview of both authentication methods, see [AWS Authentication](/appstore/connectors/aws/aws-authentication/).
+    3. In the **App Explorer**, in the **AmazonDynamoDBConnector** section, find and expand the **Connection Details** section.
+    4. Optional: If you want to use session credentials in your app, double-click **UseStaticCredentials**, and then set **Default value** to **false**.
+
+        {{< figure src="/attachments/appstore/connectors/aws-dynamodb/usestatic.png" alt="Disabling static credentials">}}
+
+    5. Depending on your selected authentication type, configure the required parameters for the **StaticCredentials** or **SessionCredentials**.
+
+        For more information about the required parameters and how to obtain them, see [AWS Authentication](/appstore/connectors/aws/aws-authentication/).
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws/aws-dynamodb.md]: Can we change this reference, this still implies that the AWS authentication docs explain how to configure it. 
Preferably we mention that the **constants** in these folders should be configured accordingly and next show the table with the constants and the explanation for it. This I think is now in the "constants" section of this doc.

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md [hunk 5]
```diff
@@ -34,21 +34,101 @@ After you install the connector, you can find it in the **App Explorer**, in the
...
+35. In the **Create Object** dialog box, click **New**.
+36. In the **Edit Change Item** dialog box, in the **Member** drop-down, select the attribute that you previously created, for example, `TableName`.
+37. In the expression editor, type `$IteratorListTable/Name`, and then click **OK**.
+
+    {{< figure src="/attachments/appstore/connectors/aws-dynamodb/microflow2.png" alt="The microflow after mapping the properties">}}
+
+38. Configure a method for triggering the **ACT_ListTables** microflow.
     For example, you can trigger a microflow by associating it with a custom button on a page in your app. For an example of how this can be implemented, see [Creating a Custom Save Button with a Microflow](/refguide/creating-a-custom-save-button/).
 
 To help you work with the Amazon DynamoDB connector, the following sections of this document list the available entities, enumerations, and activities that you can use in your application.
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws/aws-dynamodb.md:183]: It seems like the doc skips from chapter 1 to chapter 3

Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-dynamodb.md:183]: How do you mean? Chapter 2 is Installation. It's a link to another document (general instructions on installing Marketplace connectors) but it's there.

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md (hunk 4 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-dynamodb.md]: I've rewritten the section per feedback from Trong, and also made changes to the base AWS Authentication connector doc, to indicate more clearly which parts don't apply to platform-supported connectors. Can you take a look & let me know what you think?

File: content/en/docs/appstore/connectors/aws/aws-dynamodb.md (hunk 5 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-dynamodb.md:183]: N/m I noticed that the numbering was broken below the microflow config section. Fixed.

**Pull request merged**